From 915566219c4641f952d51b4c3b9f9b373e52c7bb Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 29 Nov 2024 01:03:24 +0800
Subject: [PATCH] Update imageext.c

---
 src_c/imageext.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src_c/imageext.c b/src_c/imageext.c
index f1a2b0b..0caf828 100644
--- a/src_c/imageext.c
+++ b/src_c/imageext.c
@@ -21,13 +21,6 @@
   pete@shinners.org
 */
 
-/*
- *  extended image module for pygame, note this only has
- *  the extended load and save functions, which are automatically used
- *  by the normal pygame.image module if it is available.
- */
-#include "pygame.h"
-
 /* Keep a stray macro from conflicting with python.h */
 #if defined(HAVE_PROTOTYPES)
 #undef HAVE_PROTOTYPES
@@ -40,6 +33,13 @@
 #undef HAVE_STDLIB_H
 #endif
 
+/*
+ *  extended image module for pygame, note this only has
+ *  the extended load and save functions, which are automatically used
+ *  by the normal pygame.image module if it is available.
+ */
+#include "pygame.h"
+
 // PNG_SKIP_SETJMP_CHECK : non-regression on #662 (build error on old libpng)
 #define PNG_SKIP_SETJMP_CHECK
 #include <png.h>
