From 25ba390986a0722ff0c9933e5ad58aa163b61822 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 24 Nov 2024 20:18:25 +0800
Subject: [PATCH] Update openblas.pc.in

---
 openblas.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openblas.pc.in b/openblas.pc.in
index 23804f4..ff84980 100644
--- a/openblas.pc.in
+++ b/openblas.pc.in
@@ -2,6 +2,6 @@ Name: openblas
 Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version
 Version: ${version}
 URL: https://github.com/xianyi/OpenBLAS
-Libs: -L${libdir} -l${libprefix}openblas${libnamesuffix}
+Libs: -L${libdir} -lopenblas
 Libs.private: ${extralib}
 Cflags: -I${includedir}
