From 7ada638f1eb4cef8753dba7ac1bafde0b8730be0 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 14 Sep 2024 07:09:15 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index af6814a..d8ac5ed 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -4,7 +4,7 @@
 
 [build-system]
 build-backend = 'mesonpy'
-backend-path = ['.']
+#backend-path = ['.']
 requires = [
   'meson >= 0.63.3; python_version < "3.12"',
   'meson >= 1.2.3; python_version >= "3.12"',
