From 521b66a3405e06acedf17fe5e5edff1c57facd0d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 14 Sep 2024 05:57:29 +0800
Subject: [PATCH] Update __init__.py

---
 mesonpy/__init__.py | 23 ++++++++++++++++++++++-
 1 file changed, 22 insertions(+), 1 deletion(-)

diff --git a/mesonpy/__init__.py b/mesonpy/__init__.py
index 6db5b5b..8c5e550 100644
--- a/mesonpy/__init__.py
+++ b/mesonpy/__init__.py
@@ -708,7 +708,28 @@ def __init__(
         # write the native file
         native_file_data = textwrap.dedent(f'''
             [binaries]
-            python = '{sys.executable}'
+            c = ['aarch64-openwrt-linux-musl-gcc',]
+            c_ld = ['bfd',]
+            cpp = ['aarch64-openwrt-linux-musl-g++',]
+            cpp_ld = ['bfd',]
+            ar = 'aarch64-openwrt-linux-musl-gcc-ar'
+            strip = 'aarch64-openwrt-linux-musl-strip'
+            nm = 'aarch64-openwrt-linux-musl-gcc-nm'
+            pkg-config = '/home/runner/work/mosdns/mosdns/sdk/staging_dir/host/bin/pkg-config'
+            cmake = '/home/runner/work/mosdns/mosdns/sdk/staging_dir/host/bin/cmake'
+            python = '/home/runner/work/mosdns/mosdns/sdk/staging_dir/host/bin/python3'
+            [built-in options]
+            c_args = ['-Os', '-pipe', '-mcpu=generic', '-fno-caller-saves', '-fno-plt', '-fhonour-copts', '-ffile-prefix-map=/home/runner/work/mosdns/mosdns/sdk/build_dir/target-aarch64_generic_musl/pypi/meson_python-0.16.0=meson_python-0.16.0', '-Wformat', '-Werror=format-security', '-fstack-protector', '-D_FORTIFY_SOURCE=1', '-Wl,-z,now', '-Wl,-z,relro', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/usr/include', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/include', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/include/fortify',]
+            c_link_args = ['-L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/usr/lib', '-L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/lib', '-fuse-ld=bfd', '-znow', '-zrelro',]
+            cpp_args = ['-Os', '-pipe', '-mcpu=generic', '-fno-caller-saves', '-fno-plt', '-fhonour-copts', '-ffile-prefix-map=/home/runner/work/mosdns/mosdns/sdk/build_dir/target-aarch64_generic_musl/pypi/meson_python-0.16.0=meson_python-0.16.0', '-Wformat', '-Werror=format-security', '-fstack-protector', '-D_FORTIFY_SOURCE=1', '-Wl,-z,now', '-Wl,-z,relro', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/usr/include', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/include', '-I/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/include/fortify',]
+            cpp_link_args = ['-L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/usr/lib', '-L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/lib', '-fuse-ld=bfd', '-znow', '-zrelro',]
+            [host_machine]
+            system = 'linux'
+            cpu_family = 'aarch64'
+            cpu = 'generic'
+            endian = 'little'
+            [properties]
+            needs_exe_wrapper = true
         ''')
         self._meson_native_file.write_text(native_file_data)
 
