From bab3be468b986020c044b4d24c2ad23597a688ab Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 4 Sep 2024 16:38:25 +0800
Subject: [PATCH] Update Makefile.PL

---
 Makefile.PL | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 078152f..1f1f788 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -9,7 +9,7 @@ use Config;
 use Getopt::Long;
 use ExtUtils::MakeMaker;
 use Data::Dumper;
-use Devel::CheckLib 1.12;
+# use Devel::CheckLib 1.12;
 use File::Spec;
 require DBI;
 require DBI::DBD;
@@ -227,13 +227,13 @@ MSG
 
 print "Checking if libs and header files are available for compiling...\n";
 
-assert_lib(
-  LIBS => (join ' ', @libdirs, $main_lib, @libs),
-  ccflags => (join ' ', $opt->{cflags}, ($Config{usedl} ? $Config{cccdlflags} : ())),
-  ldflags => (join ' ', @libdirs, @ldflags, ($Config{usedl} ? $Config{lddlflags} : ())),
-  header => \@mysql_headers,
-  not_execute => 1,
-);
+#assert_lib(
+#  LIBS => (join ' ', @libdirs, $main_lib, @libs),
+#  ccflags => (join ' ', $opt->{cflags}, ($Config{usedl} ? $Config{cccdlflags} : ())),
+#  ldflags => (join ' ', @libdirs, @ldflags, ($Config{usedl} ? $Config{lddlflags} : ())),
+#  header => \@mysql_headers,
+#  not_execute => 1,
+#);
 
 print "Checking if correct version of MariaDB or MySQL client is present...\n";
 
