From fed14ef2f5f951f5d87989cb3745d880b5fd8dc7 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 4 Sep 2024 17:29:11 +0800
Subject: [PATCH] Update Makefile.PL

---
 Makefile.PL | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 078152f..7608bf4 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -9,7 +9,7 @@ use Config;
 use Getopt::Long;
 use ExtUtils::MakeMaker;
 use Data::Dumper;
-use Devel::CheckLib 1.12;
+# use Devel::CheckLib 1.12;
 use File::Spec;
 require DBI;
 require DBI::DBD;
@@ -227,23 +227,23 @@ MSG
 
 print "Checking if libs and header files are available for compiling...\n";
 
-assert_lib(
-  LIBS => (join ' ', @libdirs, $main_lib, @libs),
-  ccflags => (join ' ', $opt->{cflags}, ($Config{usedl} ? $Config{cccdlflags} : ())),
-  ldflags => (join ' ', @libdirs, @ldflags, ($Config{usedl} ? $Config{lddlflags} : ())),
-  header => \@mysql_headers,
-  not_execute => 1,
-);
+#assert_lib(
+#  LIBS => (join ' ', @libdirs, $main_lib, @libs),
+#  ccflags => (join ' ', $opt->{cflags}, ($Config{usedl} ? $Config{cccdlflags} : ())),
+#  ldflags => (join ' ', @libdirs, @ldflags, ($Config{usedl} ? $Config{lddlflags} : ())),
+#  header => \@mysql_headers,
+#  not_execute => 1,
+#);
 
 print "Checking if correct version of MariaDB or MySQL client is present...\n";
 
-check_lib(
-  LIBS => (join ' ', @libdirs, $main_lib),
-  ccflags => $opt->{cflags},
-  ldflags => (join ' ', @libdirs, @libs, @ldflags),
-  header => \@mysql_headers,
-  function => "\n#ifndef MYSQL_VERSION_ID\n#include <mysql_version.h>\n#endif\nreturn (MYSQL_VERSION_ID >= 40108) ? 0 : 1;\n",
-) or die <<"MSG";
+#check_lib(
+#  LIBS => (join ' ', @libdirs, $main_lib),
+#  ccflags => $opt->{cflags},
+#  ldflags => (join ' ', @libdirs, @libs, @ldflags),
+#  header => \@mysql_headers,
+#  function => "\n#ifndef MYSQL_VERSION_ID\n#include <mysql_version.h>\n#endif\nreturn (MYSQL_VERSION_ID >= 40108) ? 0 : 1;\n",
+#) or die <<"MSG";
 
 ERROR: Wrong version or unable to check version of MariaDB/MySQL include files.
 To build DBD::MariaDB it is needed MariaDB or MySQL client >= 4.1.8.
