From 72eea49e68354c99b2337d36b1a5db2b5e0a613d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 4 Dec 2024 07:32:40 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index 9e13827..9680c67 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -89,6 +89,7 @@ manylinux-aarch64-image = "manylinux2014"
 [tool.scikit-build]
 metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
 sdist.include = ["src/blosc2/_version.py"]
+cmake.args = ["-DUSE_SYSTEM_BLOSC2=ON"]
 
 [tool.setuptools_scm]
 write_to = "src/blosc2/_version.py"
