From d9a855d7e8d557190933d268ed8bfdb1e43fcfbd Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 3 Dec 2024 09:59:05 +0800
Subject: [PATCH] Update setup_configure.py

---
 setup_configure.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup_configure.py b/setup_configure.py
index 3c5fe0d..12fd50a 100644
--- a/setup_configure.py
+++ b/setup_configure.py
@@ -260,8 +260,8 @@ def _load_hdf5_lib(self, libdirs):
             default_path = 'cyghdf5-200.dll'
             regexp = re.compile(r'^cyghdf5-\d+.dll$')
         else:
-            default_path = 'libhdf5.so'
-            regexp = re.compile(r'^libhdf5.so')
+            default_path = 'libhdf5.so.310'
+            regexp = re.compile(r'^libhdf5.so.310')
 
         path = None
         for d in libdirs:
