From c068421eee94309430fe13ec489df928812e51f2 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Mon, 2 Dec 2024 22:47:30 +0800
Subject: [PATCH] Update setup_configure.py

---
 setup_configure.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup_configure.py b/setup_configure.py
index 3c5fe0d..b8e0c33 100644
--- a/setup_configure.py
+++ b/setup_configure.py
@@ -260,7 +260,7 @@ def _load_hdf5_lib(self, libdirs):
             default_path = 'cyghdf5-200.dll'
             regexp = re.compile(r'^cyghdf5-\d+.dll$')
         else:
-            default_path = 'libhdf5.so'
+            default_path = '/home/runner/work/mosdns/mosdns/sdk/staging_dir/target-aarch64_generic_musl/usr/lib/libhdf5.so'
             regexp = re.compile(r'^libhdf5.so')
 
         path = None
