From b2f07babdbb6e5a9b4874af802a2d58fbb3cfeb7 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 11 Sep 2024 11:18:39 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0511b0c..64222b0 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -61,9 +61,9 @@ ignore_errors = true
 module = "tests.*"
 ignore_errors = true
 
-[tool.meson-python.args]
-setup = ["-Dwheel=true", "-Dtests=false"]
+#[tool.meson-python.args]
+#setup = ["-Dwheel=true", "-Dtests=false"]
 
-[build-system]
-build-backend = 'mesonpy'
-requires = ['meson-python >= 0.16.0']
+#[build-system]
+#build-backend = 'mesonpy'
+#requires = ['meson-python >= 0.16.0']
