From 1f30cbd8054121e896d628a3537b434036a49528 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 13 Dec 2024 22:53:35 +0800
Subject: [PATCH] Update zlib_ngmodule.c

---
 src/zlib_ng/zlib_ngmodule.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/zlib_ng/zlib_ngmodule.c b/src/zlib_ng/zlib_ngmodule.c
index ddeef4c..583339a 100644
--- a/src/zlib_ng/zlib_ngmodule.c
+++ b/src/zlib_ng/zlib_ngmodule.c
@@ -5,7 +5,7 @@
 
 #include "Python.h"
 #include "structmember.h"         // PyMemberDef
-#include "zlib-ng.h"
+#include <zlib-ng.h>
 #include "stdbool.h"
 #include "stdint.h"
 
