From e4a46cdfbcf48085dd78c004ed8d6c2a78a086c4 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 18 Sep 2024 13:18:38 +0800
Subject: [PATCH] Update meson.build

---
 meson.build | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/meson.build b/meson.build
index be0a51e..f59769d 100644
--- a/meson.build
+++ b/meson.build
@@ -81,11 +81,11 @@ if host_machine.system() == 'os400'
 endif
 
 # Adding at project level causes many spurious -lgfortran flags.
-add_languages('fortran', native: false)
-ff = meson.get_compiler('fortran')
-if ff.has_argument('-Wno-conversion')
-  add_project_arguments('-Wno-conversion', language: 'fortran')
-endif
+#add_languages('fortran', native: false)
+#ff = meson.get_compiler('fortran')
+#if ff.has_argument('-Wno-conversion')
+#  add_project_arguments('-Wno-conversion', language: 'fortran')
+#endif
 
 if host_machine.system() == 'darwin'
   if cc.has_link_argument('-Wl,-ld_classic')
