From dda53ea1f8f07cf9436f70eee9eeb5a77f8a7a46 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 29 Nov 2024 18:49:48 +0800
Subject: [PATCH] Update lzf.c

---
 lzf.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lzf.c b/lzf.c
index bedfdb6..c1cb271 100644
--- a/lzf.c
+++ b/lzf.c
@@ -35,6 +35,7 @@
  */
 
 #include "config.h"
+#include <stdarg.h>
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
