From 08b6f80413806b5ae63cba5a3314c845b92f1a17 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 12 Nov 2024 11:33:56 +0800
Subject: [PATCH] Update Cargo.toml

---
 Cargo.toml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index d41fbd6..d8857c0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,10 +32,10 @@ categories = []
 
 [features]
 # By deafult you need to set the `LOCAL_TDLIB_PATH` environment variable to the path of the TDLib library. See CONTRIBUTING.md for more information.
-default = ["download-tdlib"]
+# default = ["download-tdlib"]
 local-tdlib = ["tdlib-rs/local-tdlib"]
-download-tdlib = ["tdlib-rs/download-tdlib"]
-pkg-config = ["tdlib-rs/pkg-config"]
+# download-tdlib = ["tdlib-rs/download-tdlib"]
+# pkg-config = ["tdlib-rs/pkg-config"]
 
 [package.metadata.system-deps]
 tdjson = "1.8.29"
