--- a/mesonbuild/__init__.py
+++ b/mesonbuild/__init__.py
@@ -639,8 +639,8 @@ def __init__(
         self._source_dir = pathlib.Path(source_dir).absolute()
         self._build_dir = pathlib.Path(build_dir).absolute()
         self._editable_verbose = editable_verbose
-        self._meson_native_file = self._build_dir / 'meson-python-native-file.ini'
-        self._meson_cross_file = self._build_dir / 'meson-python-cross-file.ini'
+        self._meson_native_file = self._build_dir / 'openwrt-build/openwrt-native.txt'
+        self._meson_cross_file = self._build_dir / 'openwrt-build/openwrt-cross.txt'
         self._meson_args: MesonArgs = collections.defaultdict(list)
         self._limited_api = False
 
