From 5dd582a5b4ad6d917ebc5105e93ee9b65d68105e Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 10 Sep 2024 00:11:29 +0800
Subject: [PATCH] Add files via upload

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2085908..66b51a5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,7 +23,7 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-cmake_minimum_required(VERSION 3.20)
+cmake_minimum_required(VERSION 3.19)
 # XXX using 0.1.90 instead of 0.2.0-DEV
 project(nghttp3 VERSION 1.4.0)
 
